lib/collection/src/shards/replica_set/update.rs

```diff
@@ -432,10 +446,12 @@ impl ShardReplicaSet {
                 continue;
             };
 
-            if peer_state != ReplicaState::Active && peer_state != ReplicaState::Initializing {
-                continue;
+            match peer_state {
+                ReplicaState::Active | ReplicaState::Initializing | ReplicaState::Resharding => (),
+                _ => continue,
             }
 
+            // TODO(resharding): 🤔
```

Comment:
Can we skip this for resharding, because our _driver_ arranges proper synchronization?